Source/UI/NewUI/BillingForm/ViewController/BillingFormViewController.swift

```diff
@@ -64,6 +66,9 @@ final class BillingFormViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+      oldOrientation = UIKit.UIDevice.current.orientation
+      UIKit.UIDevice.current.setValue(currentOrientation.rawValue, forKey: "orientation")
+      UIFont.loadAllCheckoutFonts
```

Comment:
Why does this VC have to `loadAllCheckoutFonts`? I thought that was happening once at some earlier point?